src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/søknad/domain/SøknadBarn.kt

```diff
@@ -18,5 +19,12 @@ data class SøknadBarn(
 data class BarnMedBarnepass(
     val type: TypeBarnepass,
     val startetIFemte: JaNei?,
+    val utgifter: Utgifter?,
     val årsak: ÅrsakBarnepass?,
 )
+
+data class Utgifter(
+    val harUtgifterTilPass: JaNei,
```

Comment:
Tror denne burde renames til: 
```suggestion
    val harUtgifterTilPassHelePerioden: JaNei,
```